cosmoz-default-tree.html

```diff
@@ -15,142 +27,236 @@
 	 *
 	 * @constructor
 	 */
-	Cosmoz.DefaultTree = function (treeData) {
+	Cosmoz.DefaultTree = function (treeData, childProperty, searchProperty) {
 		Cosmoz.Tree.apply(this, arguments);
 		this._treeData = treeData;
 		this._roots = _objectValues(treeData);
+		this.childProperty = childProperty || 'children';
+		this.searchProperty = searchProperty || 'name';
 	};
 
 	Cosmoz.DefaultTree.prototype = Object.create(Cosmoz.Tree.prototype);
 
-	Cosmoz.DefaultTree.prototype.getNodeByProperty = function (propertyName, propertyValue) {
+	/** TODO (jalal): Replace with findNode()?
```

Comment:
Well I don't think it would make any sense _not_ to do it with `exact` on - such use cases can probably use `searchNodes()` and handle the results.
And it wasn't an option we had before this PR either.